edx_name_affirmation/api.py

```diff
@@ -212,3 +212,38 @@ def update_is_verified_status(
         )
     )
     log.info(log_msg)
+
+
+def create_verified_name_config(user, use_verified_name_for_certs=None):
+    """
+    Create verified name configuration for the given user.
+    Arguments:
+        * `user` (User object)
+        * `use_verified_name_for_certs` (bool): If True, certificates will prioritize
+          the user's verified name over their profile name.
+    """
+    fields = {'user': user}
+    if use_verified_name_for_certs is not None:
+        fields['use_verified_name_for_certs'] = use_verified_name_for_certs
+
+    config_obj = VerifiedNameConfig.objects.create(**fields)
+
+    log_msg = (
+        'Created verified name config for user_id={user_id} with '
+        'use_verified_name_for_certs={use_verified_name_for_certs}'.format(
+            user_id=config_obj.user.id,
+            use_verified_name_for_certs=config_obj.use_verified_name_for_certs,
+        )
+    )
+    log.info(log_msg)
+
+
+def should_use_verified_name_for_certs(user):
+    """
+    Returns a boolean describing whether the user has opted to use their verified
+    name over their profile name for certificates.
+    Arguments:
+        * `user` (User object)
+    """
+    config_obj = VerifiedNameConfig.current(user)
```

Comment:
What if the config_obj doesn't exists? Basically, the user never setup the config object?

Reply:
Looks like the non-existing is already tested. If you can explain here, that'd be great. If not, I trust this works.

Reply:
If the there is no `current` config, an empty entry (that does not persist) is returned here. This is the relevant code in the django-config-models repo https://github.com/edx/django-config-models/blob/21c0bc4bb30e5225cd490c2ec0ba1dc3b0249b78/config_models/models.py#L126

Reply:
Ah, so the `config_obj.user_verified_name_for_certs` will always return false when its empty. I guess this is the magic of config model.